packages/saga-router/README.md

```diff
@@ -151,7 +153,7 @@ Construct a new resource for loading data from a single (or dynamic) endpoint
 2. `resourceKlass` *(Object)*: Resource backend class used for providing [resource API](https://github.com/thorgate/tg-resources/tree/master/README.md#resource-api) for this endpoint.
 3. `config` *(Object)*: Object containing config for this resource. see [Configuration](#configuration)
 
-All methods returns Sagas that returns api response.
+All methods return `call` effect that returns api response.
```

Comment:
wording, this seems better: `All methods return `call` effects which yield api response.`